src/Microsoft.Security.Utilities.Core/WellKnownRegexPatterns.cs

```diff
@@ -17,95 +19,66 @@ public static class WellKnownRegexPatterns
-        new AzureContainerRegistryIdentifiableKey(),
-        new NuGetApiKey(),
-        new AdoLegacyPat(),                         // SEC101/102
-        new AzureCosmosDBLegacyCredentials(),       // SEC101/104
-        new AzureStorageAccountLegacyCredentials(), // SEC101/106
-        new AzureMessagingLegacyCredentials(),
-        new AzureDatabricksPat(),
-        new AzureEventGridIdentifiableKey(),
-    };
-
-    public static IEnumerable<RegexPattern> DataClassification { get; } = new RegexPattern[]
-    {
-        new IPv4(),
-        new IPv6(),
-        new Float(),
-        new Integer(),
-        new GuidValue(),
-    };
-
-    public static IEnumerable<RegexPattern> HighConfidenceThirdPartySecurityModels { get; } = new List<RegexPattern>
-    {
-        new NpmAuthorKey(),
-        new SecretScanningSampleToken(),
-    };
+    public static IReadOnlyList<RegexPattern> UnclassifiedPotentialSecurityKeys { get; } =
+        CreateList(new UnclassifiedJwt(),
+                   new UrlCredentials(),
+                   new LooseSasSecret(),
+                   new OAuth2BearerToken(),
+                   new Unclassified32ByteBase64String(),
+                   new Unclassified64ByteBase64String(),
+                   new AadClientAppLegacyCredentials34(),      // SEC101/101 legacy generated passwords.
+                   new Pkcs12CertificatePrivateKeyBundle(),
+                   new Unclassified16ByteHexadecimalString()
+    );
+
+    public static IReadOnlyList<RegexPattern> HighConfidenceMicrosoftSecurityModels { get; } =
+        CreateList(new UnclassifiedLegacyCommonAnnotatedSecurityKey(),
+                   new AadClientAppIdentifiableCredentials(),
+                   new AzureFunctionIdentifiableKey(),
+                   new AzureSearchIdentifiableQueryKey(),
+                   new AzureSearchIdentifiableAdminKey(),
+                   new AzureRelayIdentifiableKey(),
+                   new AzureEventHubIdentifiableKey(),
+                   new AzureServiceBusIdentifiableKey(),
+                   new AzureIotHubIdentifiableKey(),
+                   new AzureIotDeviceIdentifiableKey(),
+                   new AzureIotDeviceProvisioningIdentifiableKey(),
+                   new AzureStorageAccountIdentifiableKey(),
+                   new AzureCosmosDBIdentifiableKey(),
+                   new AzureBatchIdentifiableKey(),
+                   new AzureMLWebServiceClassicIdentifiableKey(),
+                   new AzureApimIdentifiableDirectManagementKey(),
+                   new AzureApimIdentifiableSubscriptionKey(),
+                   new AzureApimIdentifiableGatewayKey(),
+                   new AzureApimIdentifiableRepositoryKey(),
+                   new AzureCacheForRedisIdentifiableKey(),
+                   new AzureContainerRegistryIdentifiableKey(),
+                   new NuGetApiKey(),
+                   new AdoLegacyPat(),                         // SEC101/102
```

Comment:
>SEC101/102 [](http://example.com/codeflow?start=66&length=10)

i'm not happy with our partial rendering of the opaque ids, but you didn't create this issue you just moved it about. :)

Reply:
I don't like it either, I'm going to go with the majority and remove them all.